src/Core/Types/Types/Descriptors/Conventions/ConventionRecord.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HotChocolate.Types.Descriptors
```

Comment:
@michaelstaib  This class is only for registering conventions in the service collections. As we need to make a link between an interface and a implementation of said interface we probably need some wrapper